packages/flutter/lib/src/rendering/box.dart

```diff
@@ -2710,6 +2971,9 @@ abstract class RenderBox extends RenderObject {
   @override
   void debugPaint(PaintingContext context, Offset offset) {
     assert(() {
+      if (debugCheckIntrinsicSizes) {
+        _debugVerifyDryBaselines();
+      }
```

Comment:
This feels slightly out of place here since it is not painting anything? Why is this the best place for it and not where we do the other intrinsics consistency checks?

Reply:
I moved the "// Only perform the baseline checks after performLayout." paragraph here.